designs/administrator/4.4/quay-imagemanifestvuln-list/index.md

```diff
@@ -0,0 +1,29 @@
+---
+parent: Administrator
+version: 4.4
+---
+
+# Quay Image Manifest Vulnerabilities List
+
+Users that have installed the Container Security Operator now have direct access to known container image vulnerabilities in their cluster via the new **Image Manifest Vulnerabilities** nav item in the Administration section. This nav item lists the vulnerabilities on the cluster and each can be viewed to see specific details and the pods that are affected by it. Users can also access this list filtered by pod from a pod's actions menu.
```

Comment:
Is this page also accessible from the Cluster Overview's status card popover? Maybe in the future?

Reply:
Maybe these "Image Manifest Vulnerabilities" are something very different than "Image Vulnerabilities" though, as Cameron wishes we could call them. I'm likely missing a good bit of context here. 🤷‍♂️

Reply:
I don't think there is a difference @andybraren and we do wish to call them Image Vulnerabilities :) I think Peter is still trying for that! Regarding the link from the dashboard card, we actually are linking to the image manifest vulnerabilities list from the dashboard card now in 4.4. Is that what you mean?

Reply:
Ah, thank you for clarifying. I just tried it on a dev cluster and yeah, clicking "2 namespaces" in the dashboard's popover now takes you to this new nav item's list. Maybe the last sentence in this paragraph could mention this pathway as well, like:

> Users can also access this list filtered by pod from a pod's actions menu, or filtered by namespace from the Cluster Overview's Quay status popover.

Reply:
Sounds good to me, included.